draft-barnes-acme.md

```diff
@@ -1500,6 +1500,30 @@ validation of domain names.  If ACME is extended in the future to support other
 types of identifier, there will need to be new Challenge types, and they will
 need to specify which types of identifier they apply to.
 
+## Authorized Keys Objects
+
+Several of the challenges in this document makes use of an "authorized keys"
+object.  Such an object is a JSON array of objects, where each object encodes an
+authorization for a specific account key to fulfill a specific challenge.
+
+token (required, string):
+: The "token" field from the challenge object
+
+key (required, JWK):
+: The account key being authorized
+
+~~~~~~~~~~
+[
+  {
+    "token": "evaGxfADs6pSRb2LAv9IZf17Dt3juxGJ-PCt92wr-oA",
+    "key": /* account key, as a JWK object */
+  }
```

Comment:
In case of `[{token: A, key: K1}, {token: B, key: K2}]`... which key do I use for the JWS? :O In other part of the spec we require that "JWS MUST use the Flattened JSON Serialization", which implies one signature (and key) per JWS. Actually, do we ever sign this object?